packages/loader/container-loader/src/container.ts

```diff
@@ -980,7 +982,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
      *   - otherwise, version sha to load snapshot
      * @param pause - start the container in a paused state
      */
-    private async load(specifiedVersion: string | null | undefined, pause: boolean) {
+    private async load(specifiedVersion: string | null | undefined, pause: boolean, pendingOps?) {
         if (this._resolvedUrl === undefined) {
```

Comment:
When we load container with some pendingOps, we need to ensure that delta manager connects only as "write".
This is due to a fact that we need to ensure we had a chance to process all remote ops (similar to regular state with local unacked changes) to ensure we had a chance to observe our own ops.

Take a look at this code (like ~524) in DeltaManager.ts:
```
        if (this.clientSequenceNumberObserved !== this.clientSequenceNumber) {
            requestedMode = "write";
        }
```

We essentially need somehow to communicate to it that we need to do the same here.